src/App.js

```diff
@@ -4,22 +4,36 @@ import TodoList from "./TodoList";
 import AddTodoForm from "./AddTodoForm";
 
 
+const useSemiPersistentState = () =>{
+
+    const [todoList, setTodoList] = React.useState(
+        JSON.parse(localStorage.getItem('savedTodoList') || 'clear' )
```

Comment:
What does 'clear' do?

I used `JSON.parse(localStorage.getItem('savedTodoList') || '[]' )` in my code. Running 

`JSON.parse(localStorage.getItem('savedTodoList') || 'clear' )` in the javascript console gives me an error. What happens for you when you delete the savedTodoList from local storage and reload the page?